src/background/modules/AutocorrectHandler.js

```diff
@@ -236,7 +236,7 @@ export async function init() {
 
     // Thunderbird
     // Remove if part 3 of https://bugzilla.mozilla.org/show_bug.cgi?id=1630786#c4 is ever done
```

Comment:
Is this bug report/commend still correct though? LIkely not as `composeScripts` is now there?

Reply:
Yes, the comment is about declaring compose scripts in the `manifest.json` file (like with content scripts), which is not yet supported by Thunderbird. (Probably should file a new bug.)